docroot/themes/humsci/humsci_basic/src/scss/tools/_functions.color-pairings.scss

```diff
@@ -1,11 +1,19 @@
-// Map through all color pairings to find and set a color palette
-@function hb-get-pairing-color($key, $palette, $theme-map) {
+@function hb-get-pairing-list($theme-map, $palette) {
```

Comment:
can you add a comment to the top of this mixin? and I was thinking it might make more sense to name the variables like so:

```
@function hb-get-pairing-color-list($pairing-list,$palette) {
.....
```